addon/validations/factory.js

```diff
@@ -430,7 +430,7 @@ function createTopLevelPropsMixin(validatableAttrs) {
 function getCPDependentKeysFor(attribute, validations, owner) {
   let dependentKeys = validations.map(validation => {
     const type = validation._type;
-    const options = validation.options;
+    const options = getWithDefault(validation, 'options', Ember.Object.create());
```

Comment:
validation.options is fine here. This is dealing with the original validations pojo passed in by the user.